namui/src/namui/system/font/mod.rs

```diff
@@ -52,13 +52,13 @@ pub fn with_fallbacks(font: Arc<Font>) -> Vec<Arc<Font>> {
         .chain(std::iter::once_with(|| get_fallback_fonts(font_size)).flatten())
         .collect::<Vec<_>>()
 }
-pub fn get_font_of_typeface(typeface: Arc<Typeface>, font_size: IntPx) -> Arc<Font> {
+pub fn get_font_of_typeface(typeface: &Typeface, font_size: IntPx) -> Arc<Font> {
```

Comment:
I hope this function keep getting arc instead of reference. I had a problem about this before.

Reply:
What kind of problem did you have?

Reply:
The case when they want to copy reference and store it to somewhere.
`&T` cannot be stored, `Arc<T>` can be stored.